help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -38,7 +38,11 @@ Go to the [!DNL Dynamics 365] [trial](https://dynamics.microsoft.com/en-us/dynam
 #### Create field within [!DNL Dynamics 365] {#prerequisites-custom-field}
 
 Create the custom field of type `Simple` with field data type as `Single Line of Text` which Experience Platform will use to update the audience status within [!DNL Dynamics 365].
-Refer to the [!DNL Dynamics 365] documentation to [create a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need additional guidance.
+
+Refer to the [!DNL Dynamics 365] [Create or edit a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) documentation if you need additional guidance.
+
+The part of the name of any custom field you create in [!DNL Dynamics 365] is the **[!UICONTROL Customization prefix]**. Note down this value as it is required during the [Fill in destination details](#destination-details) step. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details.
```

Comment:
"The part of the name of any custom field you create"

I don't really understand what this means. Can you please rephrase this to be clearer?

Reply:
Its the same phrasing as given by MSD so as to not create confusion.

Reply:
The explanation is too convoluted and I doubt a new reader understand what that is.
Let's rephrase this section as such:

"Write down the [!UICONTROL Customization prefix] of the custom field that you create in [!DNL Dynamics 365]. You will need this prefix during the [Fill in destination details](#destination-details) step. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details."